python/xoscar/core.pyx

```diff
@@ -586,3 +587,41 @@ cdef class BufferRef:
 
     def __repr__(self):
         return f'BufferRef(uid={self.uid}, address={self.address})'
+
+
+cdef class FileObjectRef:
+    """
+    Reference of a file obj
+    """
+    _ref_to_fileobjs = weakref.WeakValueDictionary()
+
+    def __init__(self, str address, bytes uid):
+        self.uid = uid
+        self.address = address
+
+    @classmethod
+    def create(cls, fileobj: AioFileObject, address: str, uid: bytes) -> "FileObjectRef":
+        ref = FileObjectRef(address, uid)
+        cls._ref_to_fileobjs[ref] = fileobj
+        return ref
+
+    @classmethod
+    def get_file_object(cls, ref: "FileObjectRef") -> AioFileObject:
```

Comment:
```suggestion
    def get_local_file_object(cls, ref: "FileObjectRef") -> AioFileObject:
```

Reply:
Fix. Other places need call this func. So I just fix this in a new commit.